src/Backend/Modules/Settings/Actions/Index.php

```diff
@@ -643,10 +643,14 @@ private function validateForm(): void
                     'show_consent_dialog',
                     $this->form->getField('show_consent_dialog')->getChecked()
                 );
+                $privacyConsentLevels = [];
+                if($privacyConsentLevelsField->isFilled()) {
```

Comment:
codestyles also dictate that there needs to be a space after if

Reply:
fixed in 556978323d3cf8a84eeddf9db3bc4985d4696522